core/system_metric/sys_metric_stat.go

```diff
@@ -176,6 +177,10 @@ func retrieveAndUpdateCpuStat() {
 		return
 	}
 
+	// fix getProcessCpuStat return value (in percentage) breaks compatibility.
+	cpuNum := runtime.NumCPU()
```

Comment:
Is it for CPU requests or CPU limits under container env?

Reply:
Hi @sczyh30,  NumCPU returns the number of logical CPUs usable by the current process. **https://pkg.go.dev/runtime#NumCPU**.
During testing, it was found that the value is consistent with the CPU limits

Reply:
How this works in container?